reference-architecture/aws-ansible/playbooks/roles/prerequisites/tasks/main.yaml

```diff
@@ -11,3 +11,4 @@
       - bash-completion
       - docker
       - python-httplib2
+  when: ansible_pkg_mgr != "unknown" 
```

Comment:
shouldn't need ansible_pkg_mgr here at all with the package module, it should handle it on it's own on the backend.